com.unity.render-pipelines.high-definition/Runtime/VFXGraph/Shaders/VFXCommon.hlsl

```diff
@@ -146,7 +146,15 @@ float VFXSampleDepth(float4 posSS)
 
 float VFXLinearEyeDepth(float depth)
 {
-    return LinearEyeDepth(depth,_ZBufferParams);
+    if (IsPerspectiveProjection())
```

Comment:
Wasnt this fixed (and move already) in other PR?

Reply:
Ah yes, I did that change before this other PR. I can remove this change, I don't use it anymore anyways